go.mod

```diff
@@ -152,10 +151,14 @@ replace github.com/stretchr/testify => github.com/stretchr/testify v1.1.4-0.2016
 
 replace github.com/prometheus/common => github.com/prometheus/common v0.9.1
 
-replace github.com/uber-go/atomic => go.uber.org/atomic v1.6.0
+// Fix legacy import path - https://github.com/uber-go/atomic/pull/60
+replace github.com/uber-go/atomic => github.com/uber-go/atomic v1.4.0
```

Comment:
Should be fine, but why the downgrade?

Reply:
`go build` and `go get` fail otherwise:
```
go.uber.org/atomic@v1.5.0 used for two different module paths
(github.com/uber-go/atomic and go.uber.org/atomic)
```
The correct way of doing this is described in upstream issue:
> After digging into this more, per section 4.4 of the resolution section
> of How can I resolve "parsing go.mod: unexpected module path" (..),
> the correct method of resolving this is to downgrade the legacy import
> path to a version prior to the use of Go modules.

Reply:
s/issue/PR/

Reply:
Ah ok cool, thanks for the explanation!